Cargo.toml

```diff
@@ -72,16 +72,16 @@ memorydb = ["aes", "hkdf", "pbkdf2"] # for encryption
 sqlitedb = [ "dep:rusqlite", "aes", "hkdf", "pbkdf2" ]
 nssdb = [ "dep:rusqlite", "aes", "hmac", "pbkdf2" ]
 
-default = [ "sqlitedb" ]
+default = [ "standard" ]
 
 ecc_all = ["ecdsa", "ec_montgomery", "eddsa", "ecdh"]
 hash_all = ["hash", "hmac"]
 kdf_all = [ "hkdf", "pbkdf2", "sp800_108", "sshkdf", "tlskdf" ]
 
-standard = [ "ecc_all", "hash_all", "kdf_all", "rsa"]
+standard = [ "sqlitedb", "ecc_all", "hash_all", "kdf_all", "rsa" ]
 
 ecc_fips = ["ecdsa", "ecdh"]
-fips = [ "rusqlite/bundled", "aes", "ecc_fips", "hash_all", "kdf_all", "rsa"]
+fips = [ "aes", "ecc_fips", "hash_all", "kdf_all", "rsa"]
```

Comment:
nit: Do we have style rule if we should keep whitespace before closing square brace? I see half of the cases have, the other does not and you add one in the `standard` case. It would be good to have this systematic

Reply:
We never set style rules for toml, I wonder if there is a simple tool for enforcing